test/e2e_test.go

```diff
@@ -386,7 +386,40 @@ func sendRequest(config *rest.Config, url string) (*http.Response, error) {
 
 func noop() {}
 
-func consumeCPU(client clientset.Interface, podName string) (*corev1.Pod, error) {
+func watchPodReadyStatus(client clientset.Interface, podNameSpace string, podName string, resourceVersion string) error {
```

Comment:
Tests for metric accuracy already wait for metric, what's the benefit of adding a watch for pod ready?

Reply:
Let me take a look again, and analysis if there are some  difference between them.